frontend/server/src/Controllers/Course.php

```diff
@@ -242,15 +245,33 @@ private static function validateBasicCreateOrUpdate(
             }
         }
 
+        \OmegaUp\Validators::validateOptionalStringNonEmpty(
+            $r['admission_mode'],
+            'admission_mode'
+        );
+
+        if (is_null($r['admission_mode'])) {
```

Comment:
con el cambio de arriba, esto debería ser posible cambiarlo por
```suggestion
        if (
            is_null($r['admission_mode']) ||
            $r['admission_mode'] === self::ADMISSION_MODE_PUBLIC
        ) {
```

Reply:
Listo. Sólo que tuve que cambiar la condición para que sólo cuando fuera distinto a público ya no hiciera ninguna validación extra